_shared/project/tests/conftest.py

```diff
@@ -0,0 +1,70 @@
+{% if cookiecutter.get("postgres") == "yes" %}
+from os import environ
+
+{% endif %}
+import pytest
+{% if cookiecutter.get("postgres") == "yes" %}
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from {{ cookiecutter.package_name }}.db import Base
+from tests.factories.factoryboy_sqlalchemy_session import (
+    clear_factoryboy_sqlalchemy_session,
+    set_factoryboy_sqlalchemy_session,
+)
+{% endif %}
+{% if cookiecutter.get("postgres") == "yes" %}
+
+
+@pytest.fixture(scope="session")
+def db_engine():
+    engine = create_engine(environ["DATABASE_URL"])
+
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
```

Comment:
Recreate the DB schema from the models at the beginning of each test run (not each individual test). This makes sure the DB schema is always up to date with the current state of the models whenever the tests are run.